### DISCUSSION TITLE
Galactic Domination

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#8478] [Q&A] [answered]
When you dominate an entire faction do they stop being aggressive to you? If I dominate all of the Syndicate or Republic planets, will the faction ships be aggressive to my fleet?

**Comment by mOctave** [comment, marked as answer]
The faction ships will still be aggressive to your fleet, yes. They'll pay you tribute, but that won't stop them from hating your guts and trying to kill you whenever they get a chance.

If you want to dominate all the Syndicate and Republic planets, here's some advice:
- Don't bother with the planets in the Dirt Belt, Rim, etc. – anywhere that has the militia guarding it instead of the navy will cause the Free Worlds to also be hostile to you, and you want to take jobs from the Free Worlds.
- Conquer as much of the Republic as possible during the FW-Republic war. Your reputation gets reset later, and so you can keep the tribute without having to deal with aggressive ships.
- Support the Syndicate defector and start the Reconciliation branch.
- Conquer the Syndicate before the end of the war. Once again, reputation will be reset at the end of the storyline.
